src/functions.php

```diff
@@ -243,12 +249,17 @@ function dissolveGroups(array $vcards): array
  */
 function filter(array $vcards, array $filters): array
 {
+    $vcf = '';
+    foreach ($vcards as $vcard) {
+        $vcf = $vcf . $vcard->serialize();
+    }
+    file_put_contents('TEST.txt', $vcf);
+
     // include selected
     $includeFilter = $filters['include'] ?? [];
 
-    if (countFilters($includeFilter)) {
+    if (countFilters($includeFilter) > 0) {
```

Comment:
In dem Fall entspricht >0 dem true

Reply:
siehe unten

BTW
`countFilters()` liefert `int` und nicht `bool` zurück. Also immer 0 oder >0
Dann sollte die Abfrage m.E. auch darauf sich beziehen, weil ein einfaches
`if (countFilters($var))` (wie verwendet immer true zurückliefert.